docs/e2e-test.md

```diff
@@ -51,8 +51,8 @@ make clean -C /opt/metal3-dev-env/metal3-dev-env/ && sudo rm -rf /opt/metal3-dev
 
 The e2e tests currently include three different sets:
 
-1. pivoting_based_feature_tests
-1. remediation_feature_tests
```

Comment:
Why rename these? AFAIK there are multiple tests in each so it makes sense to call them `tests` rather than `test`.

Reply:
TBH I don't remember if I had a real issue with `_tests.go`, or if just Goland showed a mix of normal and test files in that directory because of the different suffixes, by convention all test files in Go should have the `_test.go` suffix. And most of them contain multiple tests.

Reply:
Good point, thanks!

Reply:
np 🙂